components/dashboard/cells/OrganizationCell.tsx

```diff
@@ -0,0 +1,31 @@
+import { ActionListAction } from '../dashboard.types';
+import styled from 'styled-components';
+
+interface Props {
+  action: ActionListAction;
+}
+
+const OrgLogo = styled.img`
+  display: block;
+  width: ${(props) => props.theme.spaces.s200};
+  height: ${(props) => props.theme.spaces.s200};
+`;
+
+const OrganizationCell = ({ action }: Props) => {
+  if (!action.primaryOrg) {
+    return null;
+  }
+
+  return (
+    <OrgLogo
+      src={
+        action.primaryOrg.logo?.rendition?.src ??
+        '/static/themes/default/images/default-avatar-org.png'
```

Comment:
This might need to have the `basePath` prefix to work properly when we are deploying under a path prefix.

Reply:
This is old code, but it could be the case that the primaryOrg logo has never been null so this hasn't been discovered